review.md

```diff
@@ -40,6 +41,7 @@ If a pull request is flagged for manual review, an `r-releases` moderator will r
 1. The text file name must be the name of the package.
 1. For JSON listings, the `"branch"` field must be `"*release"`, the `"subdirectory"` field must be supplied and exist, the `"url"` field must exist and be correct, and the `"package"` field must agree with the name of the text file.
 1. The package name, URL, and all other metadata must be complete and correct.
-1. The URL must be the true/official location of the source code or a faithful mirror of the true location. Unofficial or unsupported forks should not be included. The moderator must use discretion on a case-by-case basis because sometimes a fork becomes the true version (e.g. if the original maintainer abandons a package and becomes unreachable indefinitely).
+1. The URL must be the true/official location of the source code or a faithful mirror of the true location. The package maintainers have the authority to choose the URL. Unofficial or unsupported forks should not be included. The moderator must use discretion on a case-by-case basis because sometimes a fork becomes the true version (e.g. if the original maintainer abandons a package and becomes unreachable indefinitely).
+1. The URL must have a release on GitHub or GitLab so R-universe can process the package without an error. The package maintainers may either choose to either create their own releases (recommended) or submit the package to CRAN and use the releases from the CRAN mirror at https://github.com/cran.
```

Comment:
This wording is ok. But I would prefer a stronger formulation such as: "as a last resort, if the maintainer does not provide their own releases, releases from the CRAN mirror at ... may be submitted." The current wording sounds almost as if we're indifferent.

Reply:
Your call - I'm fine either way.

Reply:
Great point, edited in https://github.com/wlandau/r-releases-help/commit/2e53029cc7252354e10afe47f0d6261675d5326b.